services/web/server/src/simcore_service_webserver/payments/_api.py

```diff
@@ -168,14 +168,24 @@ async def complete_payment(
     await notify_payment_completed(app, user_id=transaction.user_id, payment=payment)
 
     if completion_state == PaymentTransactionState.SUCCESS:
-        # notifying RUT
-        # TODO: connect with https://github.com/ITISFoundation/osparc-simcore/pull/4692
-        settings = get_settings(app)
-        assert settings.WEBSERVER_RESOURCE_USAGE_TRACKER  # nosec
-        if base_url := settings.WEBSERVER_RESOURCE_USAGE_TRACKER.base_url:
-            url = URL(f"{base_url}/v1/credit-transaction")
-            body = (jsonable_encoder(payment, by_alias=False),)
-            _logger.debug("-> @RUTH  POST %s: %s", url, body)
+        await add_credits_to_wallet(
+            app=app,
+            product_name="",
+            wallet_id=1,
+            wallet_name="",
+            user_id=1,
+            user_email="",
+            osparc_credits=Decimal(100),
+            payment_transaction_id=payment_id,
+            created_at=datetime.now(tz=timezone.utc),
+        )
```

Comment:
I guess here is where you are faking part of the wallet selection.
My take from offline, have this "faking of wallet selection and consumed credits" be enabled via an env var.